src/test/java/com/datadog/api/v1/client/api/UsageMeteringApiTest.java

```diff
@@ -189,6 +191,69 @@ public void getUsageBillableSummaryTest() throws ApiException, IOException {
         assertEquals(logsIndexedSum.getPercentageInAccount().doubleValue(), 0.9, 0);
     }
 
+    @Test
+    public void getSpecifiedDailyCustomReportsTest() throws ApiException {
+        String reportID = "2020-07-15";
```

Comment:
@anthonypalmeira can you use `now` instead? The date used for recording is frozen in https://github.com/DataDog/datadog-api-client-java/pull/374/files#diff-a2b9b071607917a68c0392f7baa9e55fR1